S3/English/ASS_25.ass

```diff
@@ -0,0 +1,130 @@
+Dialogue: 0,0:02:43.08,0:02:45.13,Default,,0,0,0,,You're so shiny
+Dialogue: 0,0:02:48.90,0:02:49.58,Default,,0,0,0,,What's that
+Dialogue: 0,0:02:49.96,0:02:50.77,Default,,0,0,0,,I've never had one
+Dialogue: 0,0:02:51.64,0:02:52.43,Default,,0,0,0,,Let me show you
+Dialogue: 0,0:02:52.89,0:02:54.45,Default,,0,0,0,, {\pos(970,200)}It's both fun here and there
+Dialogue: 0,0:02:54.93,0:02:56.00,Default,,0,0,0,, {\pos(970,200)}Let's go to have some meal first
+Dialogue: 0,0:02:53.64,0:02:55.41,Default,,0,0,0,,They fit so well together
+Dialogue: 0,0:02:55.68,0:02:56.67,Default,,0,0,0,,I can't even have a word
+Dialogue: 0,0:02:59.19,0:03:00.59,Default,,0,0,0,,Still relying on me
+Dialogue: 0,0:03:02.46,0:03:03.63,Default,,0,0,0,,You three inside please
+Dialogue: 0,0:03:04.67,0:03:05.42,Default,,0,0,0,,Our signature dish Stinky Tofu
+Dialogue: 0,0:03:05.80,0:03:06.12,Default,,0,0,0,,Please enjoy
+Dialogue: 0,0:03:06.17,0:03:06.69,Default,,0,0,0,,I've got the bill
+Dialogue: 0,0:03:06.91,0:03:07.58,Default,,0,0,0,,Evan have some more
+Dialogue: 0,0:03:09.15,0:03:10.59,Default,,0,0,0,,Make fool of him publicly when possible
+Dialogue: 0,0:03:10.90,0:03:12.84,Default,,0,0,0,,This will make you look a bit better
+Dialogue: 0,0:03:14.18,0:03:14.60,Default,,0,0,0,,This...
+Dialogue: 0,0:03:15.08,0:03:15.75,Default,,0,0,0,,This taste...
+Dialogue: 0,0:03:16.32,0:03:17.44,Default,,0,0,0,,is so delicious
+Dialogue: 0,0:03:18.55,0:03:18.93,Default,,0,0,0,,Next
+Dialogue: 0,0:03:19.18,0:03:19.89,Default,,0,0,0,,Soybean juice
+Dialogue: 0,0:03:20.42,0:03:20.88,Default,,0,0,0,,Nice
+Dialogue: 0,0:03:21.20,0:03:21.87,Default,,0,0,0,,A dish of preserved eggs
+Dialogue: 0,0:03:21.94,0:03:22.34,Default,,0,0,0,,Yummy
+Dialogue: 0,0:03:22.48,0:03:23.00,Default,,0,0,0,,Snail Noodles
+Dialogue: 0,0:03:23.00,0:03:23.50,Default,,0,0,0,,Awesome
+Dialogue: 0,0:03:23.75,0:03:24.59,Default,,0,0,0,,May I have a takeout
+Dialogue: 0,0:03:25.11,0:03:25.76,Default,,0,0,0,,I'm full
+Dialogue: 0,0:03:25.11,0:03:25.85,Default,,0,0,0,, {\pos(970,200)}A takeout for snail noodles
+Dialogue: 0,0:03:25.91,0:03:26.44,Default,,0,0,0,,Go on
+Dialogue: 0,0:03:30.88,0:03:31.47,Default,,0,0,0,,Hurry up
+Dialogue: 0,0:03:31.47,0:03:31.65,Default,,0,0,0,,Alright
+Dialogue: 0,0:03:38.24,0:03:39.70,Default,,0,0,0,,Let's go there next
+Dialogue: 0,0:03:41.59,0:03:41.95,Default,,0,0,0,,Lily
+Dialogue: 0,0:03:42.09,0:03:42.46,Default,,0,0,0,,Wait for me
+Dialogue: 0,0:03:43.97,0:03:44.26,Default,,0,0,0,,Neil
+Dialogue: 0,0:03:44.68,0:03:45.24,Default,,0,0,0,,You alright?
+Dialogue: 0,0:03:45.60,0:03:46.31,Default,,0,0,0,,I'm fine
+Dialogue: 0,0:03:46.47,0:03:47.17,Default,,0,0,0,,Thanks
+Dialogue: 0,0:03:48.30,0:03:48.78,Default,,0,0,0,,Neil
+Dialogue: 0,0:03:50.32,0:03:51.44,Default,,0,0,0,,If Lily couldn't live happily here
+Dialogue: 0,0:03:52.44,0:03:53.89,Default,,0,0,0,,I must take her back to heaven
+Dialogue: 0,0:03:54.57,0:03:55.68,Default,,0,0,0,,This was my thought before I came here
```

Comment:
Reordered these three lines, is it better?

Reply:
Looks better! maybe "can't" isntead of couldn't and "That's what I thought before I came here" (and "But I don't think that anymore" for the next line)